### ISSUE TITLE
403 Error on workflow overview/settings within admin

### ISSUE THREAD

**Issue by charliestrafe** [engram-design/Workflow#151]
**Description**
I have a craft user who belongs to a 'reviewers' group that cannot view the workflow area. 
This is also the same for my publishers group as well, despite having the 'workflow' access ticked within both of the user group settings.

<img width="841" alt="Screenshot 2022-07-01 at 14 09 31" src="https://user-images.githubusercontent.com/47243887/176913205-c271b2c9-202f-43cf-8ce8-6b45be3725a0.png">

<img width="808" alt="Screenshot 2022-07-01 at 14 09 18" src="https://user-images.githubusercontent.com/47243887/176913218-e90471ac-ac32-4d24-becc-58684c9804c5.png">

<img width="1432" alt="Screenshot 2022-07-01 at 14 08 55" src="https://user-images.githubusercontent.com/47243887/176913242-880defd3-f0a3-4834-8445-be9eafa060bf.png">

**Steps to reproduce**
1. Create a user
2. Assign them user group which has 'workflow' access ticked
3. Try to access the workflow overview / settings area

The only workaround at the moment is to make the same user a craft 'Admin'

**Additional info**
- Plugin version: 2.0.0-beta.4
- Craft version: Craft Pro 4.1.0.2
- Multi-site: No

Thanks!

**Comment by engram-design** [comment]
Fixed for the next release. To get the fix early, change your `verbb/workflow` requirement in `composer.json` to:

```
"require": {
  "verbb/workflow": "dev-craft-4 as 2.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by charliestrafe** [comment]
I installed the new release and this is now working. thanks!

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.5](https://github.com/verbb/workflow/releases/tag/2.0.0-beta.5)
